src/style.js

```diff
@@ -105,7 +105,7 @@ const style = css`
     padding-top: 100%;
   }
   .mmp__bg {
-    background: var(--ha-card-background, var(--paper-card-background-color, white));
+    background: var(--ha-card-background, var(--card-background-color, white));
```

Comment:
Looks good, but we should keep `paper-card-background-color` as a fallback imo, to keep the card backwards compatible.
Would otherwise break in themes and HA versions whom still rely on the "old" variable.

Reply:
You're right. Fixed it.